app/lib/account/backend.dart

```diff
@@ -340,6 +340,14 @@ class AccountBackend {
     await cache.userSessionData(sessionId).purge();
   }
 
-  // TODO: periodically remove expired sessions from datastore and cache
+  /// Removes the expired sessions from Datastore and Redis cache.
+  Future deleteObsoleteSessions() async {
+    final query = _db.query<UserSession>()
+      ..filter('expires <', DateTime.now().toUtc());
```

Comment:
nit: Maybe wait for 15 min... just to account for possible clock skew...

(I just generally do this whenever I do anything with timestamps and expiration to reduce risk of bugs)